package.json

```diff
@@ -72,5 +72,9 @@
         "rollup-plugin-visualizer": "^5.9.2",
         "serve": "^14.2.0",
         "tslib": "^2.5.3"
+    },
+    "dependencies": {
```

Comment:
are these `devDependencies`?

Reply:
nope, these are part of the source

Reply:
oh interesting. what do they provide at runtime?

Reply:
So these do the object segmentation at runtime. I generally keep `devDependancies` as build tools etc, and anything used in the code itself in dependancies, but let me know what works best

Reply:
If the app builds+runs with both of these in devDependencies, they should live there.